.gitignore

```diff
@@ -1,3 +1,3 @@
 .terraform/
-README.MD
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
add possible all terragrunt and terraform folder and files in gitignore file

DS_store file should be ignore if it's any directory

Reply:
I has already put a DS_store file in . gitignore file